src/theme.ts

```diff
@@ -12,19 +12,22 @@ const palette = {
     // green colors
     main: "#006778",
     light: "#c2dbdf",
+    lighter: "#c2dbdf",
```

Comment:
Do the `light` and `lighter` have the same colour ?

Reply:
My bad, this should be fixed now.